### ISSUE TITLE
[DEP0166] DeprecationWarning

### ISSUE THREAD

**Issue by ojvribeiro** [iconify/iconify#283]
I started receiving this warning when I updated to latest Vue and Nuxt versions:

```
(node:21672) [DEP0166] DeprecationWarning: Use of deprecated double slash resolving ".//index.cjs" for module request ".//index.cjs" matched to "./*" in the "exports" field module resolution of the package at C:\Users\[...]\Documents\Projects\[...]\node_modules\@iconify\vue\package.json imported from C:\Users\[...]\Documents\Projects\[...]\node_modules\nitropack\node_modules.
```

I'm not sure if it is a @iconify/vue thing.

**Comment by cyberalien** [comment]
Yes, I've seen similar issues related to import ending in `/` in Nuxt after updating dependencies.

In my case it was fixed by removing `/` from import, so  replaced `import { stuff } from 'package/src/'` with `import { stuff } from 'package/src'`.

Maybe you have an import that ends with `/`?

**Comment by cyberalien** [comment]
Also, for Nuxt I recommend using `iconify-icon` package, not `@iconify/vue`. See https://iconify.design/docs/iconify-icon/#ssr

**Comment by ojvribeiro** [comment]
> Also, for Nuxt I recommend using `iconify-icon` package, not `@iconify/vue`. See https://iconify.design/docs/iconify-icon/#ssr

Oh, I didn't know about this one. Thank you very much.

**Comment by KarelVendla** [comment]
When running e2e tests I always got this error.
I moved away from iconify/vue (the nuxt-icon package) to iconify-icon.

I import it via the min.js in the nuxt.config.ts
```
  vue: {
    compilerOptions: {
      isCustomElement: (tag) => tag === "iconify-icon",
    },
  },
  app: {
    head: {
      script: [
        {
          src: "https://code.iconify.design/iconify-icon/2.0.0/iconify-icon.min.js",
          key: "iconify-icon",
        },
      ],
    },
  },
```
The errors are gone. thank you @cyberalien for the recommendation.

**Comment by ojvribeiro** [comment]
@KarelVendla yeah, it seems the `iconify-vue` package is throwing the same warning on build. It seems I'll have to stick to the CDN for now.

**Comment by cyberalien** [comment]
> l

Are you using it as part of nuxt icon package or separately?

**Comment by ojvribeiro** [comment]
@cyberalien basically, I created a custom <Icon> component which uses `iconify-icon` behind the scenes. I just tried to import it in the component like this:

```vue
<script setup lang="ts">
  import 'iconify-icon' // this causes build warnings
</script>

<template>
  <iconify-icon icon="..." />
</template>
```

**Comment by cyberalien** [comment]
Very strange. I'm importing it like that in Vue project too, no issues.

Can you post versions of vite, node and whatever package manager are you using?

**Comment by ojvribeiro** [comment]
Yeah, of course! This is the info generated by `nuxt info`:

```
------------------------------
- Operating System: Windows_NT
- Node Version:     v18.19.0
- Nuxt Version:     3.9.3
- CLI Version:      3.8.4
- Nitro Version:    2.8.1
- Package Manager:  yarn@1.22.19
- Builder:          -
- User Config:      extends, modules, imports, routeRules, app, googleFonts, tailwindcss, devtools
- Runtime Modules:  nuxt-typed-router@3.5.1, @vee-validate/nuxt@4.12.4, @nuxtjs/device@3.1.1, nuxt-og-image@3.0.0-rc.38
- Build Modules:    -
------------------------------
```

**Comment by cyberalien** [comment]
I've seen reports that removing `shamefully-hoist=true` from `.npmrc` fixes that.

**Comment by DbPtZn** [comment]
My project relies on the “Immer” package and the same error warning occurred

**Comment by cyberalien** [comment]
In version 5 of component removed wildcard import.

While this issue affects only Windows, so it is a Node bug, it can be annoying, so it is worth fixing. I don't think wildcard import is even used, so this shouldn't break anything. In case it does break stuff, new version is released as beta using `next` tag.

You can install it as `@iconify/vue@next`
